lib/pymedphys/_streamlit/utilities/config.py

```diff
@@ -53,6 +53,15 @@ def get_export_directory_from_site_config(site_config, export_directory):
 
 @st.cache
 def get_site_directories():
+    """A config wrapper that retrieves a dictionary that maps site to directory.
+
+    Returns
```

Comment:
typing would be very useful here, because reading the name of the function, I would think that I would get a dictionary mapping a site name to a directory.  The docstring clarifies that isn't the case.  But this is in a utilities package, even if it is intended to be internal to pymedphys streamlit app development, so it needs to be quite clear what the developer is going to get back and how to use it.  In particular, until the config.toml is documented so that the "what is getting returned" is documented in that, it's important that what comes back and how it gets used be as clearly defined as possible. 

from typing import Dict

def get_site_directories()->Dict[str, Dict[str, Any]]:

or better yet have Any replaced with the function type reference (I'm not familiar with exactly how that's done, so I punted to Any...).

Reply:
Yup, good call. Will do :).

Reply:
> or better yet have Any replaced with the function type reference (I'm not familiar with exactly how that's done, so I punted to Any...).

So, it took me quite a bit to work out how to do it, but I figure it'll be quite helpful in the future, so it was worth the time investment. But I managed to restrict the output type to something that is more descriptive.

Let me know your thoughts :).